### DISCUSSION TITLE
Static file server inaccessible after `cargo install --path .`

### DISCUSSION THREAD

**Discussion by hengheng0haha** [tokio-rs/axum#1504] [Q&A] [answered]
### Version
<img width="256" alt="image" src="https://user-images.githubusercontent.com/8019065/197983565-74c3d1fe-4f1e-48b9-923a-49519f266982.png">

### Platform
macos 13.0

### Description

Static file server inaccessible after build(`cargo install --path .`) and run. But accessible after `cargo run`.

I tried this code:
```rust
let serve_dir = ServeDir::new("./dist");
let serve_dir = get_service(serve_dir).handle_error(handle_error);
let app = Router::with_state(state)
    .route(
        "/otherapi",
        get(some_handler),
    )
    .fallback_service(serve_dir)
    .layer(TraceLayer::new_for_http());
let addr = SocketAddr::from(([0, 0, 0, 0], 8080));
info!("listening on {}", addr);
axum::Server::bind(&addr)
    .serve(app.into_make_service())
    .await?;
```

dist:  
<img width="272" alt="image" src="https://user-images.githubusercontent.com/8019065/197986764-4a5df5c6-14a7-4e8c-99c7-9baefd1f2e44.png">

after `cargo install --path .` and curl: 
`curl http://localhost:8080`
<img width="1322" alt="image" src="https://user-images.githubusercontent.com/8019065/197999805-877d4ae8-37e4-42f4-a2dc-be6239d395d8.png">

after `cargo run` and curl: 
<img width="1312" alt="image" src="https://user-images.githubusercontent.com/8019065/198000579-7191ff8b-4865-4677-941d-43cc07cbe5ec.png">

**Comment by davidpdrsn** [comment, marked as answer]
Are you running the binary from the same directory? The paths are relative to where you run it.

**Reply by hengheng0haha** [reply]
@davidpdrsn Oh! Thank you! I resolved it!
